src/panels/config/repairs/dialog-repairs-issue.ts

```diff
@@ -60,19 +82,14 @@ class DialogRepairsIssue extends LitElement {
               this._issue.translation_placeholders
             )}
           ></ha-markdown>
-
-          ${this._issue.breaks_in_ha_version
-            ? html`
-                <br />This will no longer work as of the
-                ${this._issue.breaks_in_ha_version} release of Home Assistant.
-              `
-            : ""}
-          <br />The issue is ${this._issue.severity} severity.<br />We can not
-          automatically repair this issue for you.
           ${this._issue.dismissed_version
             ? html`
-                <br />This issue has been dismissed in version
-                ${this._issue.dismissed_version}.
+                <br /><span class="dismissed">
```

Comment:
Can we make this a `ha-alert`?

Reply:
We already have one ha-alert. Adding another seems weird.